cli/azd/internal/appdetect/spring_boot.go

```diff
@@ -379,6 +393,26 @@ func IsValidDatabaseName(name string) bool {
 	return re.MatchString(name)
 }
 
+func detectPropertySpringCloudStreamBindingDestination(azdProject *Project, springBootProject *SpringBootProject) {
+	result := getBindingDestinationMap(springBootProject.applicationProperties)
+	for key, value := range result {
+		newKey := fmt.Sprintf("spring.cloud.stream.bindings.%s.destination", key)
+		azdProject.Metadata.BindingDestinationInProperty[newKey] = value
+	}
+}
+
+func detectPropertyEventhubsCheckpointStoreContainer(azdProject *Project, springBootProject *SpringBootProject) {
+	result := make(map[string]string)
+	for key, value := range springBootProject.applicationProperties {
+		if strings.HasSuffix(key, "spring.cloud.azure.eventhubs.processor.checkpoint-store.container-name") {
+			result[key] = value
+		}
+	}
+	if len(result) != 0 {
+		azdProject.Metadata.EventhubsCheckpointStoreContainer = result
```

Comment:
How about move `EventhubsCheckpointStoreContainer` from `azdProject.Metadata` to `azdProject.AzureDeps`, so that we ca avoid detect one property twice. And `ContainerNamePropertyMap` is a better name.

Same to `BindingDestinationInProperty`.

Reply:
Agree that, will update in next PR. Thanks!